meta-balena-common/recipes-core/initrdscripts/initramfs-framework_%.bbappend

```diff
@@ -110,6 +110,7 @@ SUMMARY:initramfs-module-migrate = "OS Migration"
 RDEPENDS:initramfs-module-migrate = " \
     util-linux-findmnt \
     resin-init-flasher \
+    util-linux-mountpoint \
```

Comment:
Did you mean to re-add this after it was moved in the previous commit to the base package?

Reply:
Yes, this is intended because migrate  uses `mountpoint ${FLASH_BOOT_MOUNT}` now